debian/rules

```diff
@@ -102,9 +102,9 @@ endif
 	    -DBUILD_CONFIG=mysql_release \
 	    -DCONC_DEFAULT_CHARSET=utf8mb4 \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
-			-DPLUGIN_COLUMNSTORE=NO \
+	    -DPLUGIN_COLUMNSTORE=NO \
 	    -DIGNORE_AIO_CHECK=YES \
-	    -DWITH_URING=yes \
+	    -DWITH_URING=YES \
```

Comment:
Probably not this fight but should we start using `BOOL:TRUE` and not YES.

Reply:
True, but then we should change the whole list. This at least helps align capitalization in the items.